frappe/query_builder/terms.py

```diff
@@ -48,6 +48,8 @@ def get_sql(
 		param_wrapper: Optional[NamedParameterWrapper] = None,
 		**kwargs: Any,
 	) -> str:
+		if isinstance(self.value, str) and "%" in self.value:
+			self.value = self.value.replace("%", "%%")
```

Comment:
why is this required?

This is adding an extra "%" in the query.

Before:
<img width="917" alt="Screenshot 2022-02-03 at 2 47 19 PM" src="https://user-images.githubusercontent.com/13928957/152314413-cdad50aa-18ef-4874-8183-5d9b5e7d706e.png">

with this PR:
<img width="917" alt="Screenshot 2022-02-03 at 2 47 43 PM" src="https://user-images.githubusercontent.com/13928957/152314438-4bdf5185-5035-4923-bf7a-c79cbe4555d4.png">

Reply:
That was the intent. Anyway, seems like #15857 might be fixing it so I took it out from here.